sdk/appconfiguration/app-configuration/test/public/tracing.spec.ts

```diff
@@ -45,5 +45,7 @@ describe("supports tracing", () => {
         "AppConfigurationClient.deleteConfigurationSetting",
       ]
     );
+    await client.setReadOnly({ key: key }, false);
+    await client.deleteConfigurationSetting({ key: key });
```

Comment:
This might throw if the setting got deleted by the previous call, right?
let's wrap this with a try-catch and not do anything for the "not found" error.

Reply:
Done!